src/dynamics/video_player/player/states.js

```diff
@@ -292,13 +295,30 @@ Scoped.define("module:VideoPlayer.Dynamics.PlayerStates.LoadError", [
         _started: function() {
             this.dyn.set("message", this.dyn.string("video-error"));
             this.listenOn(this.dyn, "message:click", function() {
-                this.next("Initial");
+                this.next("LoadErrorClicked");
```

Comment:
Hi @ibnutri, instead of creating a new state we can just trigger the error here. 🙂
```suggestion
                this.dyn.trigger("error:reloadplayer");
                this.next("Initial");
```